worker/restore_map.go

```diff
@@ -470,6 +472,15 @@ func (m *mapper) processReqCh(ctx context.Context) error {
 				}
 				return nil
 			}
+
+			isVectorSupportingPred := func(pred string) bool {
```

Comment:
Make it a function in schema. I think we are using something similar there also.

Reply:
In the schema, we are doing a different thing, not the same as above